gms/factories/src/main/java/com/linkedin/gms/factory/common/KafkaEventProducerFactory.java

```diff
@@ -1,4 +1,4 @@
-package com.linkedin.common.factory;
+package com.linkedin.gms.factory.common;
```

Comment:
Do you think we can use `com.linkedin.metadata.factory.common` here instead - similar to `com.linkedin.metadata.resources` namespace? Similarly for others.

Would love to hear your thoughts!

Reply:
since this is very GMS-specific rather than part of GMA-core, I feel the namespace is more appropriate?

Reply:
Sure, sounds good.